app/views/pages/home.html.erb

```diff
@@ -1,3 +1,13 @@
+
+
+<%= render "shared/banner"%>
```

Comment:
I propose to put everything what is displayed on the page in a div class container.

Reply:
Not everything otherwise the banner background-image wouldn't  be able to take the whole width.

Reply:
No, this is not true. Imagine having a parent div class home-container, next there is a div class banner-container with a background image inside. The background-image will take the whole width of it's parent container by default. Only when using margins or paddings for any of these, it will not take the whole width. 

I would be surprised if I was wrong. But I'm open for surprises. :)

Reply:
Yeah it would take the full width of its parent, the container, but the container doesn't have a width of the whole screen.

The idea of a container is that it contains the code inside a block that is smaller than the screen to focus the content of the page more toward the center with a margin auto
So if you were to have the background image inside the container you would have something like this
![image](https://github.com/wJoenn/YogaUnalome/assets/75388869/eee20490-3ddf-42b2-b5c5-17692736ed5b)

What we want instead is to have a div.banner that takes the full width of the body and inside that div.banner we add a container to center the content
![image](https://github.com/wJoenn/YogaUnalome/assets/75388869/d1dc448c-ff7a-4c61-8af6-8dee918d7313)


That means that in the code it would look something like
```
body
  div.banner
    div.container
      ...
      ...
    /div
  /div
  
  div.container
    main
  /div
/body
```

Reply:
So imagine:

`<body>
  <div class="container-page-content">
    <div class="container-banner">
    </div>
  </div>
</body>`

or

`<body>
  <div class="container-banner">
  </div>
  <div class="container-page-content">
  </div>
</body>`

container-banner would have the full width of container-page-content in the first example, which has the full width of body, unless no different styles appplied.

in second example container-banner and container-page-content would have the same width as body if not styled different.

The idea of that divs is, to structure content, to control layout, style and maybe js-behaviour on point and of course to increase readability. That's why whenever I was doing something on that pages, I wrapped everything in divs.

The graphics you used do not show the real behaviour (if not styled with css like this), they are just to show the structure and relation between body and containers.

In the second example the banner is wrapped in the body. If you don't size the body width with css, it has the whole screen width. same if you wrap a container (which contains the banner) in the body and don't size it with css it has the whole width by default.

That is my understanding. What else should be the default size of a container wrapped in the body if no style is applied?

Reply:
I didn't change anything as it looks good so far. If we dont like smth in the future we can try to do it

Reply:
Sorry, tried to make my position understandable, but it did not show the code, that's why I deleted my text, copied it before, but it got lost, now I need to go. :D :D :D 

But as I've seen, you read it. 

To follow a structure, put everything in divs, that was the idea. But we can leave it like this and finally merge. :)

"Everything that should not be wider than 1200px or 80% should be contained in a .container like this one, the membership cards for example are inside of it because we don't want the cards to take the width of the whole screen"

Okay, this is an interesting logic. Sounds good.